### ISSUE TITLE
Form field but no documentation

### ISSUE THREAD

**Issue by kobadg** [statamic/v2-docs#187]
I went to use the form fieldset, however I cannot find out how, as there is no documentation.

**Comment by jackmcdade** [comment]
Did you check out <https://docs.statamic.com/forms#create> ?

**Comment by kobadg** [comment]
Yes, however that doesn't explain the use of the fieldtype* named form. Gives a dropdown of all the forms in the formset?

Asked Jason about it, stated that it ideally will print the name of the form, he stated yes.

`{{ if formName == "contact" }}` being a scenario it would work  for. Or something along that lines.

**Comment by jasonvarga** [comment]
You can leave this open, since we need to add documentation to https://docs.statamic.com/fieldtypes/form

To clear things up, this is about the Form Fieldtype (added in [2.1.15](https://statamic.com/changelog/2.1.15)), which essentially lets you choose one or more forms.

```
the_form:
  type: form
  max_items: 1 # setting to 1 will treat this as a dropdown and save a string instead of an array
```

Then you could do something like:

```
{{ form:create :in="the_form" }} ... {{ /form:create }}
```

**Comment by seanhudson** [comment]
@jasonvarga do you have an example that I can use to see how this would work? I'm wanting to use the Form Fieldtype. Thanks.

**Comment by redolivedev** [comment]
There is still no documentation on this

**Comment by Christophvh** [comment]
Please add this somewhere in the docs:
 max_items: 1 # setting to 1 will treat this as a dropdown and save a string instead of an array

I spend hours searching for a way to select a single item.
